examples/cms-buttercms/.env.local.example

```diff
@@ -0,0 +1,2 @@
+NEXT_EXAMPLE_CMS_BUTTERCMS_API_KEY=
```

Comment:
Missing comment to explain to rename to `.env.local`.

Reply:
Done!